bevrand.frontend/assets/js/login.js

```diff
@@ -35,7 +35,7 @@ $("#loginForm").submit(function( event ) {
 function loginUser(userList) {
     $.ajax({
         type: "POST",
-        url: `${config.proxyHostname}/api/login`,
+        url: `${config.proxyHostname}/authentication-api/login`,
```

Comment:
Misschien kunnen we bij de login even kijken dat we alleen op basis van email in kunnen gaan loggen, dan worden het wel 2 calls (dus url: `${config.proxyHostname}/playlist-api/Users?by-email=${email}` ) om dan vervolgens de gehele info op te kunnen sturen.